### DISCUSSION TITLE
Does the extension as it currently exists allow for LoRA models to be applied to/in prompts?

### DISCUSSION THREAD

**Discussion by mariaWitch** [deforum-art/sd-webui-text2video#29] [Q&A]
Question is title. Considering the rather specialized nature of the models, I'm just wondering if LoRA support is already built in or if that is something that is planned for the future.

**Comment by kabachuha** [comment]
It's not implemented currently, yet given the nature of these models are the same, it should definitely be possible. I've already seen Dreambooth attempts (it's easier as it just replaces the whole model), Lora and other tuning methods will require a little bit more work

**Reply by patrickjonesdotca** [reply]
Are there any docs to the DB attemps? Would like to try that myself

**Reply by kabachuha** [reply]
Yes, there's this repo https://github.com/ExponentialML/Text-To-Video-Finetuning/ for Dreambooth/LoRA finetuning. And you can convert the weights and use in the extension. I updated this repo's README to include the info on fine-tunes and their usage https://github.com/deforum-art/sd-webui-text2video#fine-tunes-and-how-to-use-them
